File: lifespanCalc.py
totalSpan=85
compareYear=14
percentageYear=35
percentageLeft=0;


def calcLifespan (totalSpan, compareYear):
    subjectiveYear=1.0/compareYear
    total=0.0
    totalAtPercentageYear=0.0
    i=1
    while i < totalSpan:
        total+=1/i
        print(1/i)
        if(i==percentageYear):
            totalAtPercentageYear=total
        i+=1
    
    print(total)
    print(totalAtPercentageYear/total)
    print(total/subjectiveYear)
    return (total/subjectiveYear)


calcLifespan(totalSpan,compareYear)

File: lifespanCalc.js
/** 
let totalSpan=125;
let compareYear=18;
const calcLifespan = (totalSpan, compareYear) =>{
    let subjectiveYear=1/compareYear
    let total=0;
    for(let i = 0; i<totalSpan; i++){
        total+=1/i;
        console.log(1/i);
    }
    console.log(total);
    console.log(total/subjectiveYear);
}
calcLifespan(totalSpan,compareYear);
*/